examples/angular-1/wizard/components/three/three.html

```diff
@@ -1,5 +1,5 @@
 <section>
-  <h2>{{three.question}}</h2>
+  <h2>{{ three.question }}</h2>
```

Comment:
my $0.02: for a reason of consistency with other examples, I'd leave it without extra spaces